apps/snippets/models.py

```diff
@@ -95,10 +95,10 @@ class NavigationMenu(ClusterableModel):
         max_length=255,
         null=False,
         blank=False,
-        help_text=('Call the menu "header" to use as upper menu or '
-                   '"footer" to use in the lower dark grey footer. '
-                   'To display the upper light grey footer, call the '
-                   'menu snippet “content-footer”.')
+        help_text=('Call the menu "header" to use as upper menu. '
+                   'Call the menu “content-footer” to use '
+                   'the upper light grey footer, the lower '
+                   'footer is loaded externally.')
```

Comment:
not sure on the wording and didn't it would trigger a migration so maybe we should check with MK first to make sure we don't have to redo?

Reply:
The old wording also wasn't checked. But maybe it's a good idea to do some renaming. Will see if I can add a migration to rename the old content-footer to footer!